build/index.js

```diff
@@ -0,0 +1,2 @@
+require('./roundProgress.min');
```

Comment:
A couple of very minor nitpicks here: 

1. Can you add ".js" in the `require`. Technically it's not necessary, but I prefer having it.
2. Can you add semicolon and a newline after `module.exports = 'angular-svg-round-progressbar'`.